tests/test_needtable.py

```diff
@@ -76,6 +76,32 @@ def test_doc_needtable_options(test_app):
 
     assert column_order in html
 
+    # Checks for needs_string_links in needtable
+    assert "EXAMPLE_STRING_LINKS" in html
+    assert (
+        '<a class="reference external" href="https://github.com/useblocks/sphinxcontrib-needs/issues/404">'
+        "GitHub #404</a>" in html
```

Comment:
I miss the check for the second value `303` here.

Reply:
> I miss the check for the second value `303` here.

Changes have been effected.